src/pmdas/gfs2/pmns

```diff
@@ -92,6 +93,12 @@ gfs2.glstats {
     journal		GFS2:2:9
 }
 
+gfs2.glockfd {
+    total		GFS2:7:0
+    process		GFS2:8:0
+    file_descriptor	GFS2:8:1
```

Comment:
we've tended to use the short-hands "fds" in other places, e.g.

pmcd.openfds
openmetrics.grafana.process_open_fds
openmetrics.grafana.process_max_fds

Similarly, we could more descriptively use "pids" instead of "process" here too.